src/longopt/project_inc.h

```diff
@@ -25,53 +25,26 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'A', "azimuth",           "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'C', "center|origin",     "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'E', "endpoint",          "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'F', "outformat|outvars|convention", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "step|generate",
 	          "",                  "",
+	          "c,h,n",             "colat,header,npoints",
 		  GMT_TP_STANDARD },
-	{ 0, 'Q', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'S', "",
-	          "",                  "",
+	{ 0, 'L', "length",
+	          "w",                 "limit",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'T', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'W', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'Z', "",
-	          "",                  "",
+	{ 0, 'N', "flat_earth",        "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'Q', "mixed_units",       "", "", "", "", GMT_TP_STANDARD },
```

Comment:
Man says `Specify that , , r, s are in degrees while p, q, dist, lmin, lmax, wmin, wmax are in km`, so I still prefer `km` to `mixed_units`.